src/containers/Root.js

```diff
@@ -20,8 +20,12 @@ export default class Root extends React.Component {
   get devTools () {
     if (__DEBUG__) {
       if (__DEBUG_NEW_WINDOW__) {
-        require('../redux/utils/createDevToolsWindow')(this.props.store)
-      } else {
+        if (!window.devToolsExtension) {
+          require('../redux/utils/createDevToolsWindow')(this.props.store)
+        } else {
+          window.devToolsExtension.open()
+        }
+      } else if (!window.devToolsExtension) {
```

Comment:
The purpose of the check is to determine of we need to include the devtools in the root component or not (if the extension exists in the browser, we don't need to, because the same functionality (in fact, the devtools library) is included in the extension).

The extension is always being used if it exists (it just need to be registered as a middleware, it does not need to be included in the root component). Therefore it always overrides the local tools.